LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIMission_LWCustomMission.uc

```diff
@@ -135,10 +135,83 @@ simulated function BindLibraryItem()
 				ShadowChamber = Spawn(class'UIAlertShadowChamberPanel', LibraryPanel);
 				ShadowChamber.InitPanel('ShadowChamber');
 			}
+
 			break;
 		default:
-			super.BindLibraryItem();
-			break;
+			/*********************************** Issue #140 ***********************************
+			* The code for this default case replaces the call to super.BindLibraryItem(). It's actually just a copy/paste
+			* of the super.BindLibraryItem() function with the below 2 lines taken out:
```

Comment:
Could we not just call the `super` function here and hide the panel afterwards?

Reply:
I'm such a derp. I fixed this with my latest commit. Just put the call to super.BindLibraryItem() back in and then just called ShadowChamber.Hide() in there to hide the panel.